cve_bin_tool/cli.py

```diff
@@ -59,7 +81,15 @@ def main(argv=None):
         + "\n\nPlease disclose issues responsibly!",
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    parser.add_argument("directory", help="directory to scan")
+    parser.add_argument("directory", help="directory to scan", nargs="?", default=None)
+
+    parser.add_argument(
+        "-i",
+        "--input-file",
+        action="store",
+        default=None,
+        help="Provide input filename",
+    )
```

Comment:
New option to specify input file

Reply:
shouldn't it be the actual path to the file? I guess it should be file-path